mods/noita-mp/files/scripts/util/CustomProfiler.lua

```diff
@@ -25,9 +26,12 @@ CustomProfiler.reportDirectory           = ("%s%sNoitaMP-Reports%s%s"):format(fu
 CustomProfiler.reportFilename            = "report.html"
 CustomProfiler.reportJsonFilenamePattern = "%s.json"
 
+---@param functionName string
+---@return integer
+---@diagnostic disable-next-line: duplicate-set-field
 function CustomProfiler.start(functionName)
     if not ModSettingGetNextValue("noita-mp.toggle_profiler") then
-        return
+        return -1
```

Comment:
Why `-1`?

Reply:
I didn't change that, it must have been something from the merge commit

Reply:
Np, then it was me